src/clr-angular/forms/datepicker/datepicker.module.ts

```diff
@@ -27,7 +28,10 @@ export const CLR_DATEPICKER_DIRECTIVES: Type<any>[] = [
 ];
 
 @NgModule({
-    imports: [CommonModule, ClrHostWrappingModule, ClrConditionalModule, ClrIconModule, ClrFocusTrapModule],
+    imports: [
+        CommonModule, ClrHostWrappingModule, ClrCommonFormsModule, ClrConditionalModule, ClrIconModule,
```

Comment:
Actually I didn't know what `ClrCommonFormsModule` had. It shouldn't be imported here.